packages/mdl-ripple/animation.js

```diff
@@ -2,13 +2,31 @@
  * Create a sequential animation/transition from a list of class names.
  */
 export class SequentialAnimation {
+  /**
+   * @param {ElementRef} ref
+   * @param {!Object} renderer
+   * @param {boolean=} opt_isAnimation
+   */
   constructor(ref, renderer, opt_isAnimation = false) {
+    /** @private {ElementRef} */
     this.elementRef_ = ref;
+
     this.renderer_ = renderer;
-    this.reject_;
+
+    /** @private {string} */
     this.eventName_ = opt_isAnimation ? 'animationend' : 'transitionend';
+
+    /** @private {boolean} */
     this.isAnimation_ = opt_isAnimation;
+
+    /** @type {func()} */
     this.onFinish;
```

Comment:
maybe assign `null` here instead of just printing an expression? since we're not running closure over this, in most instances this will either be removed or just evaluated to a useless expression and v8 won't create the proper hidden class shape necessary to accommodate those properties.